spec.html

```diff
@@ -4958,19 +4959,19 @@ <h1>ToUint16 ( _argument_ )</h1>
       <p>The abstract operation ToUint16 takes argument _argument_. It converts _argument_ to one of 2<sup>16</sup> integer values in the range 0 through <emu-eqn>2<sup>16</sup> - 1</emu-eqn>, inclusive. It performs the following steps when called:</p>
       <emu-alg>
         1. Let _number_ be ? ToNumber(_argument_).
-        1. If _number_ is *NaN*, *+0*, *-0*, *+&infin;*, or *-&infin;*, return *+0*.
+        1. If _number_ is *NaN*, *+0*<sub>𝔽</sub>, *-0*<sub>𝔽</sub>, *+&infin;*, or *-&infin;*, return *+0*<sub>𝔽</sub>.
         1. Let _int_ be the Number value that is the same sign as _number_ and whose magnitude is floor(abs(_number_)).
-        1. Let _int16bit_ be _int_ modulo 2<sup>16</sup>.
+        1. Let _int16bit_ be _int_ modulo 2<sub>𝔽</sub><sup>16<sub>𝔽</sub></sup>.
```

Comment:
In places where we use superscripts to denote exponentiation, can't we do that in the mathematical space and then represent it as its Number value? So this could become 

```suggestion
        1. Let _int16bit_ be _int_ modulo 2<sup>16</sup><sub>𝔽</sub>.
```

But this particular usage has the same awkwardness that I mention above with trying to do too much in the Number space.